src/components/app/Layout.tsx

```diff
@@ -1,12 +1,19 @@
 import {FunctionComponent} from 'react'
+import dynamic from 'next/dynamic'
 import Header from './Header'
 import Footer from './Footer'
 import Main from './Main'
 
+let CommandBar: FunctionComponent = () => <></>
+
+if (process.env.NODE_ENV == 'development' && process.env.NEXT_PUBLIC_DEVTOOLS) {
+  CommandBar = dynamic(() => import('./CommandBar')) as FunctionComponent
+}
 const Layout: FunctionComponent = ({children}) => {
   return (
     <>
       <Header></Header>
+      <CommandBar></CommandBar>
```

Comment:
This can go anywhere, but it will need access to a Context/whatever to `send` commands to.

We could also show/hide with a keyboard shortcut if we want to develop this further.

Reply:
We could also conditionally render like `{devtools && <CommandBar/>}`